packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -14,6 +15,7 @@ class DatadogSpanContext {
     this._tags = props.tags || {}
     this._sampling = props.sampling || {}
     this._baggageItems = props.baggageItems || {}
+    this._tracestate = props.tracestate || new TraceState()
```

Comment:
Any way to avoid always creating an instance of `TraceState` here?

Reply:
Could probably get the tracer config and check if tracecontext is enabled each time, though I don't know if that's really any cheaper than constructing the unused map. 🤷🏻